src/Notifications/Notifications.csproj

```diff
@@ -8,7 +8,7 @@
   <PropertyGroup Condition=" '$(RunConfiguration)' == 'Notifications-SelfHost' " />
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.SignalR.Protocols.MessagePack" Version="6.0.4" />
-    <PackageReference Include="Microsoft.AspNetCore.SignalR.StackExchangeRedis" Version="6.0.4" />
+    <PackageReference Include="Microsoft.AspNetCore.SignalR.StackExchangeRedis" Version="6.0.22" />
```

Comment:
Is there a reason redis is part of self-hosted? And do we want to bump the other SignalR dependencies, I'm not familiar with if they expect to stay in sync.

Reply:
It's not really that it's a part of self-hosted it's just not explicitly dis-allowed. I suspect the number of SH using it is 0. 

No they don't need to be updated at the same time. This package just manages the connection lifetime and the other package is just for how we encode our messages.